cmd/krel/cmd/gcbmgr.go

```diff
@@ -140,8 +140,10 @@ func runGcbmgr() error {
 		logrus.Fatal("Cannot specify both the 'stage' and 'release' flag. Please resubmit with only one of those flags selected.")
 	}
 
-	gcbSubs, gcbSubsErr := setGCBSubstitutions()
-	if gcbSubsErr != nil {
+	gcbSubs := map[string]string{}
+	var gcbSubsErr error
+	gcbSubs, gcbSubsErr = setGCBSubstitutions(gcbSubs)
```

Comment:
I preferred the previous approach, is there any reason why we now pass the `gcbSubs` in?

Reply:
Heh. I did too!
I was getting nil panics because the `gcloud auth list` command was failing and `setGCBSubstitutions` was return a nil map. So here I'm basically making sure we initialize the map before hitting the function.

...but I think I can fix it up. Give me a sec.